infrastructure/main.tf

```diff
@@ -50,40 +50,28 @@ resource "azurerm_key_vault_secret" "POSTGRES_PORT" {
   key_vault_id = data.azurerm_key_vault.key_vault.id
 }
 
-resource "azurerm_key_vault_secret" "POSTGRES_DATABASE" {
-  name         = "recipe-backend-POSTGRES-DATABASE"
-  value        = module.recipe-database.postgresql_database
-  key_vault_id = data.azurerm_key_vault.key_vault.id
-}
-
 resource "azurerm_key_vault_secret" "POSTGRES-USER-V11" {
   name         = "recipe-backend-POSTGRES-USER"
```

Comment:
this is the line you want to change to append with v11 not the key vault id

Reply:
Yes my bad .. the issue was the duplicate names of the secrets.. corrected those now.